app/server/schema.graphql

```diff
@@ -1,15 +1,18 @@
 """The application data"""
```

Comment:
This file probably shouldn't be updated by this PR

Reply:
I've changed the schema generation to be in alphabetic order. Could split if needed

Reply:
[`--sort-export`](https://www.graphile.org/postgraphile/usage-cli/#cli-options)
lexicographically (alphabetically) sort exported schema for more stable diffing.